### ISSUE TITLE
`aio app init` fails when choosing a workspace that has services added

### ISSUE THREAD

**Issue by jdelbick** [adobe/aio-cli-plugin-app#242]
**Describe the bug**
When I run `aio app init` and choose a workspace that has some services added, I got an error.

**To Reproduce**
1. create a firefly project on https://console.adobe.io/
2. add "IO Events" api service to a workspace
3. run `aio app init` and choose the project and workspace above
4. see error

**Expected behavior**
No errors

**Screenshots**
<img width="649" alt="Screen Shot 2020-05-22 at 1 36 45 PM" src="https://user-images.githubusercontent.com/20048485/82710091-e1484b80-9c36-11ea-8e3e-d61e31e50a95.png">

**Desktop (please complete the following information):**
 - iOS
 - chrome

**Additional context**
This is related to a change with console api released on Thursday. The config JSON for a workspace used to be missing the techacct id:
```{
    "project": {
        ...,
        "workspace": {
            ...
            "details": {
                "credentials": [
                    {
                        "id": "91303",
                        "name": "Project 1588810146307J",
                        "integration_type": "service",
                        "jwt": {
                            "client_id": "74964c79613c42ceb6ba6af06876b29b",
                            "client_secret": "1894ed14-3566-44d3-92e7-87a98436418b",
                            "techacct": "375b1b6e-8bb3-4846-857c-9b5541c4555e@techacct.adobe.com",
                            "meta_scopes": [
                                "ent_adobeio_sdk",
                                "event_receiver_api"
                            ]
                        }
                    }
                ],
                ...
                    ]
                }
            }
        }
    }
}
```

So with the change Thursday, the new format of the JWT part is like this:
```
"jwt": {
               "client_id": "74964c79613c42ceb6ba6af06876b29b",
               "client_secret": "1894ed14-3566-44d3-92e7-87a98436418b",
               "technical_account_email": "375b1b6e-8bb3-4846-857c-9b5541c4555e@techacct.adobe.com",
                "technical_account_id": "07D356945EB351A20A495F95@techacct.adobe.com",
                "meta_scopes": [
                                "ent_adobeio_sdk",
                                "event_receiver_api"
                            ]
     }
```

So the technical account information has moved from:
```
config.project.workspace.details.credentials[0].jwt.techacct
```
to here:
```
config.project.workspace.details.credentials[0].jwt.technical_account_email 
config.project.workspace.details.credentials[0].jwt.technical_account_id
```

**Comment by shazron** [comment]
Thank you! will be fixed in #243 and released in a patch update soon

**Comment by shazron** [comment]
app plugin 2.0.8 published
please do an `aio update`.
